docs/user-guide/rules.md

```diff
@@ -62,6 +62,7 @@ Here are all the rules within stylelint, grouped by the [*thing*](http://apps.wo
 
 ### Time
 
+-   [`time-min-milliseconds`](../../lib/rules/time-min-milliseconds/README.md): Limit the minimum number of milliseconds for time values.
```

Comment:
I think "Specify" is a better word here than "Limit": "Specify the minimum number ..."